### PR TITLE
lookup: add more native modules

### PR DESCRIPTION
**Pull request by TheAlphaNerd**
* sqlite3
* iconv
* electron-prebuilt
* libxmljs
* radium
* ffi
* microtime

### CODE REVIEW

**Comment by TheAlphaNerd** [comment]
citgm: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/463/

**Comment by GeorgeAdams95** [comment]
kicking off a citgm run on our ibm machines, will let you know the outcome

File: lib/lookup.json [hunk 1]
```diff
@@ -313,5 +313,32 @@
   "full-icu-test": {
     "replace": true,
     "prefix": "v"
+  },
+  "sqlite3": {
+    "replace": true,
+    "prefix": "v"
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix

File: lib/lookup.json [hunk 2]
```diff
@@ -313,5 +313,32 @@
...
+  },
+  "iconv": {
+    "replace": true,
+    "prefix": "v"
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix

File: lib/lookup.json [hunk 3]
```diff
@@ -313,5 +313,32 @@
...
+  },
+  "electron-prebuilt": {
+    "replace": true,
+    "prefix": "v"
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix, ppc64le, s390x

File: lib/lookup.json [hunk 4]
```diff
@@ -313,5 +313,32 @@
...
+  },
+  "libxmljs": {
+    "replace": true,
+    "prefix": "v"
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix, windows

File: lib/lookup.json [hunk 5]
```diff
@@ -313,5 +313,32 @@
...
+  },
+  "radium": {
+    "replace": true,
+    "prefix": "v"
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix, ppc64le, s390x

File: lib/lookup.json [hunk 6]
```diff
@@ -313,5 +313,32 @@
...
+  },
+  "ffi": {
+    "replace": true
```
Comment (GeorgeAdams95) [inline lib/lookup.json]: flaky:aix, ppc64le, s390x

**Comment by GeorgeAdams95** [comment]
PR added to your fork: https://github.com/TheAlphaNerd/citgm/pull/2

**Comment by TheAlphaNerd** [comment]
looks like the only legitimate failure was `radium-v0.18.1` on fedora 22

**Comment by TheAlphaNerd** [comment]
landed in 433f17ce5d

**Pull request closed without merging**